templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-main.yaml

```diff
@@ -78,10 +79,12 @@ Metadata:
       # Organization parameters
       ExecutionRoleRuleName:
         default: "Name of Execution Role for Organization"
-      CloudTrailRegions:
-        default: "CloudTrail Regions of Management Account"
-
-
+      CloudTrailRegion:
+        default: "CloudTrail Region of Management Account"
+      DelegatedAdminAccountId:
```

Comment:
Do we need to add delegated adminstrator for one region only in order to collect logs from multiple regions?

Reply:
We should let the user specify a region to collect logs from multiple regions